src/client/BulkMatchClient.ts

```diff
@@ -529,7 +561,7 @@ class BulkMatchClient extends SmartOnFhirClient {
                 // Download is finished – emit event and save file off
                 this.emit("downloadComplete", {
                     fileUrl: file.url,
-                    duration: Date.now() - downloadStartTime,
+                    duration: formatDuration(Date.now() - downloadStartTime),
```

Comment:
Just a question: Isn't this (at least in case of bulk export) supposed to be a number? And are we even trying to keep this as in bulk export, or do we have different events here?

Reply:
To your point though: we _are_ diverging meaningfully from the events in bulk export. I have that on my list of things to discuss with you, but it is not a big priority right now. 😄

Reply:
Sure, we can discuss later. My rule of thumb for such cases is:
- Use user-friendly string while presenting (e.g.: print formatted date or duration in terminal)
- Use a timestamp as number in case a consumer might want to do their own formatting or otherwise consume that as raw variable rather than a presentation label

Reply:
We can probably ask for Mike's opinion as an actual consumer of such logs?